st2actions/st2actions/policies/retry.py

```diff
@@ -117,6 +122,16 @@ def apply_after(self, target):
 
         return target
 
+    def _get_live_action_parent(self, live_action_db):
```

Comment:
Perhaps `_is_live_action_part_of_workflow_action` might be a bit more clear and obvious (it would make it more generic and hide the abstraction that child workflow actions have `parent` attribute in context), but it's not a big deal.